Add BPMN MIWG 2023 demo results to reference

barmac opened the pull request:
This PR adds the 2023 demo results to the Reference directory. The models used are from [the live demo folder](https://www.dropbox.com/home/BPMN%20MIWG%20Demo%202023/2024-07-18%20Live%20Run), and the renderings were produced with [bpmn-to-image](https://github.com/bpmn-io/bpmn-to-image), equivalent to [Camunda Desktop Modeler](https://github.com/camunda/camunda-modeler/) rendering.

SimonRinguette commented on Reference/C.9.1.png:
This model has a lane that goes over the pool header. This should be corrected.

SimonRinguette commented on Reference/C.9.2.bpmn:
```diff
@@ -0,0 +1,3882 @@
+         <callActivity name="Document Request"
+                        id="_1c9f4f0d-d193-4643-8201-0843a9a937fe"
+                        isForCompensation="false"
+                        calledElement="process_1c9f4f0d-d193-4643-8201-0843a9a937fe"
```
The called element here is not defined or imported, there is no way to know if the + marker should be displayed in the diagram.

barmac replied:
I've added the imports via https://github.com/bpmn-miwg/bpmn-miwg-test-suite/pull/912/commits/5252a245255bb7d5bb5e58662ced68c1cf6c01c9

barmac commented:
@barmac to use the reference models instead

barmac commented:
I've updated the test cases to use [the reference models](https://www.dropbox.com/home/BPMN%20MIWG%20Demo%202023/2023-04-07%20Reference%20Model), and added the entries in the JSON.

SimonRinguette commented:
Make sense to me, we should merge
